aio/deploy/helm-chart/kubernetes-dashboard/README.md

```diff
@@ -14,7 +14,7 @@ helm install my-release kubernetes-dashboard/kubernetes-dashboard
 
 ## Introduction
 
-This chart bootstraps a [Kubernetes Dashboard](https://github.com/kubernetes/dashboard) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+This chart bootstraps a [Kubernetes Dashboard](https://github.com/kubernetes/dashboard) deployment on a [Kubernetes](https://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
```

Comment:
Regarding the Helm Chart, even a small change like this would need a version bump in Chart.yaml.

Reply:
done